lib/github.atd

```diff
@@ -590,7 +600,7 @@ type repo_issue_event = {
   ?label: base_label option;
   ?assignee: linked_user option;
   ?assigner: linked_user option;
-  ?milestone: milestone option;
+  ?milestone: repo_issue_milestone option;
```

Comment:
Yes it was wrong before. You just get a `title` back from this particular part of the API. See https://developer.github.com/v3/issues/events/#list-events-for-a-repository

> milestone
> The Milestone object including a `title` attribute. Only provided for `milestoned` and `demilestoned` events.

Roll on GraphQL...